src/App.js

```diff
@@ -61,6 +29,7 @@ function App() {
     setNotes([...notes,newNote])
     onCloseModal()
   }
+  localStorage.setItem('notes', JSON.stringify(notes));
```

Comment:
please remove this, we should not call localStorage like this, this will be called each time component is rendered, which means almost all the time (each time we change the state)